pages/release-notes.mdx

```diff
@@ -139,6 +139,124 @@ Memgraph (v2.11.1 and lower).
+- Fixed a bug that caused the incorrect coordinator to become the leader due to
+  missing cluster state data.
+  [#1897](https://github.com/memgraph/memgraph/pull/1897)
+- Added [automatic index creation](/fundamentals/indexes) for node labels and
+  edge types. Two new configuration flags control this feature for labels and
+  edge-types automatically encountered by the database.
+  [#1841](https://github.com/memgraph/memgraph/pull/1841)
+- Added support for `bolt+routing` when connecting to a cluster of Memgraph
+  instances. [#1796](https://github.com/memgraph/memgraph/pull/1796)
+- Added support for [edge type indices](/fundamentals/indexes#edge-type-index)
+  to optimize query performance on graphs.
+  [#1542](https://github.com/memgraph/memgraph/pull/1542)
+- Expanded the []`min()` and `max()`](/querying/functions#aggregation-functions)
+  functions to support temporal types, improving date and time manipulation
+  capabilities.
+  [#1790](https://github.com/memgraph/memgraph/pull/1790)
+- Renamed [HA flags](/configuration/configuration-settings#high-availability)
+  `--raft-server-id` to `--coordinator-id`, `--raft-server-port` to
+  `--coordinator-port`, `--coordinator-server-port` to `--management-port`
+  to improve cluster setup.
+  [#1820](https://github.com/memgraph/memgraph/pull/1820)
+  - Reduced the size of the Delta from 80B to 56B
+  [#1747](https://github.com/memgraph/memgraph/pull/1747)
+- Extended the C API to support [query
+  execution](/custom-query-modules/cpp/cpp-api#executionheaders). Extension also
+  provided on the C++ level.
+  [#1773](https://github.com/memgraph/memgraph/pull/1773)
+- [Updated queries](/configuration/high-availability) used for registering
+  replication instances and coordinator instances.
+  [#1809](https://github.com/memgraph/memgraph/pull/1809)
+- ¸[Introduced the lock on operations](/configuration/high-availability) such as
+  `SETTING INSTANCE` to MAIN, `REGISTER INSTANCE`, `UNREGISTER INSTANCE`, and
+  `FAILOVER` for better control of the cluster state.
+  [#1819](https://github.com/memgraph/memgraph/pull/1819)
+- When doing failover the Coordinator [chooses a new
+  MAIN](/configuration/high-availability) by comparing which `memgraph database`
+  is the up-to-date. [#1729](https://github.com/memgraph/memgraph/pull/1729)
+- [Changing instance's replication role](/configuration/high-availability) is
+  now backed by Raft. [#1728](https://github.com/memgraph/memgraph/pull/1728)
+- If REPLICA diverges from MAIN, MAIN will automatically initiate a forced data
+  reset on REPLICA. [#1777](https://github.com/memgraph/memgraph/pull/1777)
+- User can create, set, or remove labels using property values.
+  [#1762](https://github.com/memgraph/memgraph/pull/1762)
+- Introduced the ability to [fetch edges based on edge
+  ID](/configuration/configuration-settings#storage).
+  [#1808](https://github.com/memgraph/memgraph/pull/1808)
+- Added support for [pattern
```

Comment:
@kgolubic Whenever we have a new feature in Cypher, please review https://memgraph.com/docs/querying/differences-in-cypher-implementations and update it